library/init/meta/expr.lean

```diff
@@ -265,6 +265,9 @@ meta constant expr.get_delayed_abstraction_locals : expr → option (list name)
 @[inline] meta def reflected.to_expr {α : Sort u} {a : α} : reflected a → expr :=
 id
 
+/-- This is a more strongly-typed version of `expr.subst` that keeps track of the value being
+reflected. To obtain a term of type `reflected _`, use `` (`(λ x y, foo x y).subst ex).subst ey`` instead of
+using `` `(foo %%ex %%ey) `` (which returns an `expr`). -/
```

Comment:
Do we care about line-wrapping?

Reply:
That's up to you, I wouldn't worry about it.